### ISSUE TITLE
windIO GitHub group

### ISSUE THREAD

**Issue by kenloen** [IEAWindSystems/windIO#40]
Currently windIO is a repository on the *IEAWindTask37*, but we are going to continue the work in [Ref-Wind (IEA Wind Task 55)](https://iea-wind.org/task-55/). We could consider to move windIO in to a *IEAWindTask55* group but I would suggest to move it into dedicated windIO group?!
In that way it would also be easier collect supporting tools like converters and examples.

**Comment by fzahle** [comment]
I would actually vote to create a WindIO organisation to elevate WindIO beyond the IEA Task. Task 55 will also stop at some point, just like Task 37 did...

**Comment by kenloen** [comment]
I agree, I would much prefer that - but how and who can/will do that?

**Comment by kenloen** [comment]
At today's IEA Task 55 meeting we discussed different possibilities of changing the group for windIO as well as other parts of IEAWindTask37.

The following options were discussed (without a conclusion):

1. Change the name of IEAWindTask37. The following suggestions for the name were discussed:
   - REFWind
   - IEAWind
2. Create a group for windIO. The following suggestions were discussed:
   - windIO (There is already a GitHub group called [Windio](https://github.com/Windio), so that is likely not possible)
   - REFWind
   - IEAwindIO

**Comment by kilojoules** [comment]
Why note name the group windio? That way it will be clear that the reference systems in our group use the windio ontology.

**Comment by rafmudaf** [comment]
The organization name has changed from `IEAWindTask37` to `IEAWindSystems`. Does that resolve this issue?
